app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb

```diff
@@ -1,16 +1,19 @@
-<p>You may be eligible for a bursary or scholarship if your teacher training course leads to <%= link_to("qualified teacher status (QTS)", page_path("train-to-be-a-teacher/what-is-qts")) %>. These are tax-free amounts of money you receive to train in certain subjects. You do not need to pay them back.</p>
-
-<p>Your bursary or scholarship will be paid in a minimum of 10 equal monthly instalments over the duration of your course by your teacher training provider.</p>
-
-<p>Your teacher training provider will confirm the payment dates and amounts.</p>
-
-<p>The following amounts apply if you're starting your teacher training course between September 2024 and July 2025.</p>
+<section class="col col-720">
+  <p>This page is for people who have a degree or are studying towards one, and want to do postgraduate teacher training in England.  To help fund your training, you may be eligible for a bursary or scholarship.</p>
+  <h2>What's a bursary?</h2>
+  <p>Bursaries are tax-free payments aimed at encouraging graduates to train to teach certain subjects. If you're eligible for a bursary, you'll get it automatically.</p>
+  <h2>What's a scholarship?</h2>
+  <p>Scholarships are tax-free payments provided by charitable organisations. As well as money, you'll have the opportunity to access benefits like mentoring and networking. If you're eligible, you must apply and only a limited number of scholarships are awarded.</p>
+  <h2>What to consider</h2>
+  <p>If you're considering a bursary or scholarship, you should know:<ul><li>you cannot receive both</li><li>they're not available for salaried teacher training</li><li>only certain subjects have bursaries and scholarships but this changes every year</li><li>the figures on this page apply if you're starting your training between September 2024 and July 2025</li></p>
 
 <%= render Content::ExpanderComponent.new(
-    title: "bursaries and scholarships",
-    text: "If you train to teach languages or physics, you may be eligible for a bursary or scholarship. Learn more about <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">fees and financial support for non-UK trainee teachers</a>.",
-  ) %>
-
+  title: "bursaries and scholarships",
+  text: "If you train to teach languages or physics, you may be eligible for a bursary or scholarship. Bursaries and scholarships are not available for other subjects..",
+  link_title: "Learn more about fees and financial support for non-UK trainee teachers.",
+  link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
+) %>
+</section>
 <section id="funding-widget">
```

Comment:
Do we need to change the h2 on the funding widget to align better with the restructured page?